cmd/internal/pipeline/evaluator.go

```diff
@@ -0,0 +1,59 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"github.com/hacbs-contract/ec-cli/cmd/internal/utils"
+	"github.com/open-policy-agent/conftest/runner"
+	"github.com/spf13/afero"
+)
+
+var createWorkDir = afero.TempDir
+
+type Evaluator struct {
+	Context       context.Context
+	Target        EvaluationTarget
+	PolicySources []PolicySource
+	Paths         ConfigurationPaths
+	TestRunner    runner.TestRunner
+	Namespace     []string
+	OutputFormat  string
+	workDir       string
+}
+
+func (e *Evaluator) addDataPath() error {
+	dataDir := fmt.Sprintf("%s/data/", e.workDir)
```

Comment:
Let's use `filepath.Join`
```suggestion
	dataDir := filepath.Join(e.workDir, "data")
```